site/styles/_buttons.scss

```diff
@@ -124,6 +124,14 @@
       fill: #000;
       stroke: #000;
     }
+  }
+  &.s72-btn-trailer {
+    background: transparent;
+    border: 0 none;
```

Comment:
Curious what the `none` is doing? I thought `0` would suppress it?

Reply:
That's been merged in from master... I didn't write it/don't know what it's for.